cli.py

```diff
@@ -66,8 +66,20 @@ def cli_entry_point(scrape, resume, verify, resume_verify, compile_active, delet
         return
 
     if scrape:
-        if dm is None:
+        if not dm:
```

Comment:
While None evaluates to False, anything other than None and 0 evaluates to True. Specifying it to be None is a safe way of programming. #DefensiveProgramming